SAMv1.tex

```diff
@@ -217,7 +217,11 @@ \subsection{The header section}
 \emph{Regular expression}: \emph{name}{\tt (,}\emph{name}{\tt )*}
 where \emph{name} is {\tt [0-9A-Za-z][0-9A-Za-z*+.@\_|-]*}\\\cline{2-3}
   & {\tt AS} & Genome assembly identifier. \\\cline{2-3}
-  & {\tt M5} & MD5 checksum of the sequence in the uppercase, excluding spaces but including pads (as `*'s).\\\cline{2-3}
+  & {\tt M5} & MD5 checksum of the sequence.
+The sequence must be in the 7-bit US-ASCII character set, with pads represented as `*'s if present.
```

Comment:
The part about "pads" is unclear, I think. perhaps you can modify my example below to include "pads"?